
doing netfilter changes and turning on netfilter debug means
we've got to interpret netfilter warning messages a bit more.

---
 include/net/netfilter/nf_conntrack.h |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: linux-2.6.24.2-rt2/include/net/netfilter/nf_conntrack.h
===================================================================
--- linux-2.6.24.2-rt2.orig/include/net/netfilter/nf_conntrack.h	2008-02-21 13:10:48.000000000 -0500
+++ linux-2.6.24.2-rt2/include/net/netfilter/nf_conntrack.h	2008-02-21 13:11:26.000000000 -0500
@@ -63,11 +63,14 @@ union nf_conntrack_help {
 #ifdef CONFIG_NETFILTER_DEBUG
 #define NF_CT_ASSERT(x)							\
 do {									\
-	if (!(x))							\
+	if (!(x)) {							\
 		/* Wooah!  I'm tripping my conntrack in a frenzy of	\
 		   netplay... */					\
 		printk("NF_CT_ASSERT: %s:%i(%s)\n",			\
 		       __FILE__, __LINE__, __FUNCTION__);		\
+		if (printk_ratelimit())					\
+			WARN_ON(1);					\
+	}								\
 } while(0)
 #else
 #define NF_CT_ASSERT(x)
